CONTRIBUTING.md

```diff
@@ -57,11 +57,30 @@ Addresses #bugnumber (in this specific format)
 Your pull request should:
 
 * Include a description of what your change intends to do
+  * The title of your PR should be a very brief description of the change, and
```

Comment:
I would like to encourage an 80 character limit without being too insistent on it. If it's 90 characters, it's not worth rejecting a change over, if it's 120, we start to have a problem . . .

Reply:
I see that a few lines up it says 80 characters, but perhaps we can add the nuances I suggested here since it is a rule oft ignored.